file/readahead_raf.cc

```diff
@@ -15,15 +15,6 @@
 #include "util/rate_limiter.h"
 
 namespace rocksdb {
-
```

Comment:
`#include "file/read_write_util.h"` in this file as well?

Reply:
Fixed. Good catch.